luma/oled/const.py

```diff
@@ -39,3 +39,552 @@ class ssd1362(common):
     DISPLAYON = 0xAF
     DISPLAYOFF = 0xAE
     SETCONTRAST = 0x81
+
+
+class ws0010(object):
+    CLEAR = 0x01
+    HOME = 0x02
+    ENTRY = 0x06
+    DISPLAYOFF = 0x08
+    DISPLAYON = 0x0C
+    POWEROFF = 0x13
+    POWERON = 0x17
+    GRAPHIC = 0x08
+    CHAR = 0x00
+    FUNCTIONSET = 0x29
+    DL8 = 0x10
+    DL4 = 0x00
+    DDRAMADDR = 0x80
+    CGRAMADDR = 0x40
+    FONTDATA = {
+        # TODO: Extend and verify Unicode Mappings
```

Comment:
if it's a todo not happening in this pull request, please open a ticket and reference it here